### PR TITLE
More fixes

### PR DESCRIPTION
**Pull request by Akira25**
Some more minor fixes for buildbot.

### CODE REVIEW

File: roles/buildbot/tasks/main.yml [hunk 1]
```diff
@@ -157,16 +157,25 @@
     - packagefeed_master.pub
     - packagefeed_master.sec
 
-- name: Copy helper scripts (fw-selector)
+- name: Copy keys
   copy:
     src: "{{ item }}"
     dest: /usr/local/src
-    mode: 0700
+    mode: "0700"
```
Comment (nicolasberens) [inline roles/buildbot/tasks/main.yml:164]: we cand rop this to 0600

File: roles/buildbot/files/packages.py [hunk 2]
```diff
@@ -198,25 +198,25 @@ def packagesArchFactory(f):
...
+    #         command=["sh", "-c", util.Interpolate(
+    #             "mkdir -p %(kw:symlinksrc)s", symlinksrc=symlinksrc)]))
+    # f.addStep(
+    #     steps.MasterShellCommand(
+    #         name="symlink artifacts to url",
+    #         # might have happened, that another worker created the links already.
+    #         # That isn't a problem though
+    #         haltOnFailure=False,
+    #         command=["sh", "-c", util.Interpolate(
+    #             "ln -s %(kw:symlinkdest)s %(kw:symlinksrc)s", symlinkdest=symlinkdest, symlinksrc=symlinksrc)]))
```
Comment (nicolasberens) [inline roles/buildbot/files/packages.py:220]: lets just complete delete those lines

File: roles/buildbot/tasks/main.yml (hunk 1 above)
Comment (Akira25) [reply roles/buildbot/tasks/main.yml]: dropped with new commit.

File: roles/buildbot/files/packages.py (hunk 2 above)
Comment (Akira25) [reply roles/buildbot/files/packages.py]: deleted those lines in the new commit

**Review by nicolasberens — APPROVED** [review]

**Pull request merged**